menpo/io/input/base.py

```diff
@@ -517,7 +517,7 @@ def import_pickles(
     shuffle=False,
     as_generator=False,
     verbose=False,
-    **kwargs,
+    **kwargs
```

Comment:
All these changes are because I think your formatting is fighting with `black` - you probably want to run `black` on all the code before submitting - might be worth installing the `black` plugin for PyCharm as well

Reply:
I added all these changes for compatibility reasons with python 3.5. SInce the no reason to support python 3.5, I am more than happy to change the formatting